pandas/core/frame.py

```diff
@@ -5011,8 +5012,14 @@ def pivot(self, index=None, columns=None, values=None):
         one  1   2   3
         two  4   5   6
 
-        A ValueError is raised if there are any duplicates.
+        >>> df.pivot(index='foo', columns='bar', values=['baz', 'zoo'])
+                baz       zoo
```

Comment:
The alignment seems a bit off here. I think the 'baz' should align with the 'A' below. Best to check how it is in the console output.